pkg/resources/objects.go

```diff
@@ -13,3 +17,20 @@ func GetControllerOf(controllee meta_v1.Object) *meta_v1.OwnerReference {
 	}
 	return nil
 }
+
+// GetJsonPathString extracts the value from the object using given JsonPath template
+func GetJsonPathString(obj map[string]interface{}, path string) (string, error) {
+	j := jsonpath.New("GetJsonPathField")
+	// If the key is missing, return an empty string without errors
+	j.AllowMissingKeys(true)
+	err := j.Parse(path)
+	if err != nil {
+		return "", fmt.Errorf("JsonPath parse %s error: %v", path, err)
+	}
+	var buf bytes.Buffer
```

Comment:
It's better to declare this as a plain variable - should be allocated on the stack unlike if created via `new()`. Well, this is implementation dependent of course, but still there is no need to allocate it as a pointer.